src/app/Scenes/HomeView/Sections/MarketingCollectionsRailHomeViewSection.tsx

```diff
@@ -0,0 +1,56 @@
+import { Flex } from "@artsy/palette-mobile"
+import { MarketingCollectionsRailHomeViewSection_section$key } from "__generated__/MarketingCollectionsRailHomeViewSection_section.graphql"
+import { CardRailFlatList } from "app/Components/Home/CardRailFlatList"
+import { SectionTitle } from "app/Components/SectionTitle"
+import { MarketingCollectionRailItem } from "app/Scenes/HomeView/Sections/MarketingCollectionRailItem"
+import { extractNodes } from "app/utils/extractNodes"
+import { graphql, useFragment } from "react-relay"
+
+interface MarketingCollectionsRailHomeViewSectionProps {
+  section: MarketingCollectionsRailHomeViewSection_section$key
+}
+
+export const MarketingCollectionsRailHomeViewSection: React.FC<
+  MarketingCollectionsRailHomeViewSectionProps
+> = ({ section }) => {
+  const data = useFragment(fragment, section)
+  const component = data.component
+
+  if (!component) return null
+
+  const marketingCollections = extractNodes(data.marketingCollectionsConnection)
+  if (!marketingCollections || marketingCollections.length === 0) return null
+
+  return (
+    <Flex>
+      <Flex pl={2} pr={2}>
+        <SectionTitle title={component.title} />
+      </Flex>
+
+      <CardRailFlatList<any>
```

Comment:
fwiw, we have a node type helper that can be useful here
```suggestion
<CardRailFlatList<ExtractNodeType<MarketingCollectionsRailHomeViewSection_section["marketingCollectionsConnection"]>>
```

Reply:
Maybe we can export the type above the component